ajax/libs/easyregex/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "easyregex",
+  "description": "A library making it easier to use regular expressions",
+  "author": "George Valtas",
+  "filename": "easy-regex.min.js",
+  "version": "0.1",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/gogeorge/EasyRegex.git"
+  },
+  "keywords": [
+    "regex", 
+    "easy", 
+    "simple", 
+    "javascript"
```

Comment:
Would you delete a space after `"regex", `, `"easy", `, `"simple", ` let these keywords to fit the CDNJS's format and pass the CI?

it will look like as the followings:
```js
  "keywords": [
    "regex",
    "easy",
    "simple",
    "javascript"
  ],
```

Thank you.